packages/workbox-core/_private/cacheWrapper.mjs

```diff
@@ -38,9 +38,11 @@ const putWrapper = async (cacheName, request, response, plugins = []) => {
   let responseToCache = await _isResponseSafeToCache(
     request, response, plugins);
 
-  // TODO If response is not safe to cache - print info to log.
-
   if (!responseToCache) {
+    if (process.env.NODE_ENV !== 'production') {
+      logger.debug(`Response '${getFriendlyURL(request.url)}' is not safe ` +
```

Comment:
Can you change this message to something like "...does not meet the criteria to be cached?" instead of talking about safety? I could see a developer being freaked out if they think that something about their web traffic is unsafe.